tests/conftest.py

```diff
@@ -240,6 +247,45 @@ def minimal_attribute_dict():
     }
 
 
+@pytest.fixture()
+def minimal_dataframe_dict():
+    """links transformers to minimal dataframes needed to successfully run transformer.
+    New transformers need to be added here"""
+
+    num_df = create_numeric_df_1()
+    object_df = create_object_df()
+    date_df = create_is_between_dates_df_1()
+
+    # generally most transformers will work with num_df
+    min_df_dict = {x[0]: num_df for x in get_all_classes()}
+
+    # where they do not, will need to override
+    date_transformers = [x[0] for x in get_all_classes(wanted_module="tubular.dates")]
+    for transformer in date_transformers:
+        min_df_dict[transformer] = date_df
+
+    object_modules = ["tubular.mapping", "tubular.nominal", "tubular.strings"]
```

Comment:
N: `# override dict value with object_df for transformers in tubular.mapping, nominal and strings modules`

Reply:
updated here https://github.com/lvgig/tubular/pull/253/commits/36cc5e1244eb45a7f358fdfb4f73659ecf7b0dae